lib/internal/util.js

```diff
@@ -316,7 +316,8 @@ promisify.custom = kCustomPromisifiedSymbol;
 function join(output, separator) {
   let str = '';
   if (output.length !== 0) {
-    for (var i = 0; i < output.length - 1; i++) {
+    let i;
+    for (i = 0; i < output.length - 1; i++) {
```

Comment:
Just one nit about these: it is best to initialize `i` directly. That way there's one internal `map` type less for this variable.

Reply:
Okay, did that. Didn't do it earlier because a `for`loop without a initialization looked a bit weird at first glance.